src/fortran_routing/mc_pylink_v00/MC_singleSeg_singleTS/MCsingleSegStime_f2py_noIC_clean.f90

```diff
@@ -0,0 +1,345 @@
+module mc
+    use var
+    implicit none
+
+contains
+
+    subroutine main
+    !** This code implements Muskingum-Cunge method of NWM to a single reach that has
+    !* multiple segments (nodes)
+    !* refer to p.18-, WRF-HYDRO, RM1.
+        !open(unit=22,file="./output/qdc depth vel.txt",status="unknown")
+        !open(unit=1,file="./output/sub MC investigation1.txt",status="unknown")
+
+        call mcNWM()
+
+        !write(22,"(4F20.5)")  ql, qdc, vel, depth
+    end subroutine main
```

Comment:
Any reason to keep this?

Reply:
Commented lines are removed and call mcNWM() remains due to any future use. It can eventually removed once all are finalized.

Reply:
Great.